python/veles/util/bigint.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2017 CodiLime
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import operator
+
+from veles.compatibility.int_bytes import int_to_bytes, int_from_bytes
+
+
+HDR_BIAS = 0x80000000
```

Comment:
This seems like a little more than I hope we will ever need to use, but on the other hand we probably won't care that much about some additional bytes used by our bignums in DB and/or over network.